UPGRADE-7.0.md

```diff
@@ -55,10 +55,23 @@ Filesystem
 
  * Add argument `$lock` to `Filesystem::appendToFile()`
 
+Form
+----
+
+ * Throw when using `DateTime` or `DateTimeImmutable` model data with a different timezone than configured with the
+   `model_timezone` option in `DateType`, `DateTimeType`, and `TimeType`
+ * Make the "widget" option of date/time form types default to "single_text"
+
 FrameworkBundle
 ---------------
 
  * Remove command `translation:update`, use `translation:extract` instead
+ * Make the `http_method_override` config option default to `false`
+ * Remove `AbstractController::renderForm()`, use `render()` instead
```

Comment:
I would suggest adding a before/after example here. As it stands, I think people will use `render(..., ['form' => $form->createView()])` again (which is not want we suggest).

Reply:
added